apps/modernization-ui/src/pages/patient/profile/ProfileContext.tsx

```diff
@@ -19,9 +20,9 @@ export function useProfileContext() {
 }
 
 export const ProfileProvider = ({ children, id }: any) => {
-    const { refetch } = usePatientProfile(id);
+    const { refetch, profile } = usePatientProfile(id);
 
     const changed = () => refetch();
 
-    return <ProfileContext.Provider value={{ changed }}>{children}</ProfileContext.Provider>;
+    return <ProfileContext.Provider value={{ changed, profile }}>{children}</ProfileContext.Provider>;
```

Comment:
issue:  combining `patient` and `summary` into one object will cause the entire page to re-render.  An alternative approach would be to have `usePatientProfile`return the `patient` and `summary` as two separate items in `PatientProfile.tsx`

`const {patient, summary} = usePatientProfile()`

Reply:
got it